script.js

```diff
@@ -30,3 +30,211 @@ contactLink.addEventListener('click', () => {
   mobileMenu.classList.replace('show_menu', 'hide_menu');
   header.classList.replace('hide_menu', 'show_menu');
 });
+
+// Array of project objects to load the content dynamically
+const works = [
+  {
+    id: 'featureWork',
+    name: 'Multi-Post Stories',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard dummy text
+                  ever since the 1500s, when an unknown printer took a standard
+                  dummy text.`,
+    featImage: 'imgs/img-placeholder-fw.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://sonuengineer.github.io/MyPortfolio/',
+    sourceLink: 'https://github.com/sonuengineer/MyPortfolio',
+  },
+  {
+    id: 'work1',
+    name: 'Profesional Art Printing Data',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://sonuengineer.github.io/MyPortfolio/',
+    sourceLink: 'https://github.com/sonuengineer/MyPortfolio',
+  },
+  {
+    id: 'work2',
+    name: 'Data Dashboard Healthcare',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
+    technologies: ['html', 'javascript', 'css'],
+    liveLink: 'https://sonuengineer.github.io/MyPortfolio/',
+    sourceLink: 'https://github.com/sonuengineer/MyPortfolio',
+  },
+  {
+    id: 'work3',
+    name: 'Website Protfolio',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
+    technologies: ['html', 'javascript', 'Ruby'],
+    liveLink: 'https://sonuengineer.github.io/MyPortfolio/',
+    sourceLink: 'https://github.com/sonuengineer/MyPortfolio',
+  },
+  {
+    id: 'work4',
+    name: 'Profesional Art Data Printing',
+    description: `A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard.`,
+    featImage: '../imgs/placeholder-img.png',
+    technologies: ['html', 'css', 'Ruby'],
```

Comment:
- [ ] Currently, the path for the featImage property here is not working. Having a `../imgs` means you are moving out of a directory and into another directory. But technically, you want to go into a directory here, from the `script.js`, and hence, it should be `./imgs`. Remove one dot from this path here so the images load and display on the popup modal properly 👍